src/sql/workbench/services/connection/browser/connectionManagementService.ts

```diff
@@ -906,14 +906,15 @@ export class ConnectionManagementService extends Disposable implements IConnecti
 				}
 				const tenantId = connection.azureTenantId;
 				const token = await this._accountManagementService.getAccountSecurityToken(account, tenantId, azureResource);
-				this._logService.debug(`Got token for tenant ${token}`);
 				if (!token) {
 					this._logService.info(`No security tokens found for account`);
```

Comment:
This seems like it should be a warning - that isn't something that we expect to happen right?

Reply:
Sure

Reply:
```suggestion
					this._logService.warn(`No security tokens found for account`);
```